packages/extension/src/libs/backup-state/index.ts

```
@@ -0,0 +1,363 @@
+              signerType as SignerType,
+              i,
+            );
+            const existingAccount = getAccountByIndex(
+              existingAccounts,
+              basePath,
+              signerType as SignerType,
+              i,
+            );
+            if (existingAccount && newAccount) {
+              await kr.renameAccount(existingAccount.address, newAccount.name);
+              continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)